lib/fluent/plugin/out_forward.rb

```diff
@@ -166,8 +167,8 @@ def configure(conf)
       end
 
       if @transport == :tls
-        if @tls_cert_path && !@tls_cert_path.empty?
```

Comment:
To avoid existing environment breakage, if `tls_cert_path` is not `nil`, assign `tls_cert_path` to `tls_ca_cert_path` is better.

Reply:
Add code to keep compatibility at 71d202f.